packages/eslint-config-react-app/index.js

```diff
@@ -155,6 +155,15 @@ module.exports = {
     'unicode-bom': ['warn', 'never'],
     'use-isnan': 'warn',
     'valid-typeof': 'warn',
+    'no-restricted-properties': ['warn', {
+      object: 'require',
+      property: 'ensure',
+      message: 'Please use import() instead.'
```

Comment:
I don't believe we have docs for this yet. 🙈

@tharakawj, would you like to contribute some? 😄